eng/targets/Settings.props

```diff
@@ -30,7 +30,7 @@
     <UseRoslynAnalyzers Condition="'$(UseRoslynAnalyzers)' == ''">true</UseRoslynAnalyzers>
 
     <!-- Set to non-existent file to prevent common targets from importing Microsoft.CodeAnalysis.targets -->
-    <CodeAnalysisTargets>*none*</CodeAnalysisTargets>
+    <CodeAnalysisTargets>NON_EXISTENT_FILE</CodeAnalysisTargets>
```

Comment:
@dotnet/roslyn-infrastructure I meant to submit this as a separate pull request but I forgot to extract it. Please review this change.

Reply:
What does this change actually do? Was `*none*` a magic value before?

Reply:
There is no change in the behavior. MSBuild uses `Exists($(CodeAnalysisTargets))` for a condition. `*none*` does not exist, but unlike the new value, also contains `*` characters that are not valid. This change avoids a bunch of first chance exceptions that were making it hard for me to figure out why AnalyzerRunner was failing to load Roslyn.sln.